--- /usr/share/roundcube/program/lib/Roundcube/rcube_vcard.php	2018-10-23 13:12:54.000000000 +0200
+++ /usr/share/roundcube/program/lib/Roundcube/rcube_vcard.php2	2020-02-06 01:03:34.000000000 +0100
@@ -536,7 +536,7 @@
                 '/^item\d*\.X-AB.*$/mi',  // remove cruft like item1.X-AB*
                 '/^item\d*\./mi',         // remove item1.ADR instead of ADR
                 '/\n+/',                 // remove empty lines
-                '/^(N:[^;\R]*)$/m',      // if N doesn't have any semicolons, add some
+                '/^(N:[^;\r\n]*)$/m',      // if N doesn't have any semicolons, add some
             ),
             array(
                 '\2;type=\5\3:\4',
